packages/@stimulus/core/src/tests/modules/class_tests.ts

```diff
@@ -1,7 +1,7 @@
 import { ControllerTestCase } from "../cases/controller_test_case"
 import { ClassController } from "../controllers/class_controller"
 
-export default class ValueTests extends ControllerTestCase(ClassController) {
+export default class ClassTests extends ControllerTestCase(ClassController) {
```

Comment:
this change isn't related to this PR, it just fixes an incorrect class name

Reply:
Thank you! Fixed in main by caecf0f186b08164b6c152ce6e6ba95645a3fcda